source/addonAPIVersion.py

```diff
@@ -13,12 +13,13 @@
 """
 
 CURRENT = (buildVersion.version_year, buildVersion.version_major, buildVersion.version_minor)
-BACK_COMPAT_TO = (2021, 1, 0)
+BACK_COMPAT_TO = (2022, 1, 0)
 """
 As BACK_COMPAT_TO is incremented, the changed / removed parts / or reasoning should be added below.
 These only serve to act as a reminder, the changelog should be consulted for a comprehensive listing.
 EG: (x, y, z): Large changes to speech.py
 ---
+(2022, 1, 0): TODO: update during the dev cycle:

```

Comment:
These are the vast majority and most notable [changes found in this release](https://github.com/nvaccess/nvda/blob/master/user_docs/en/changes.t2t#L72)

```suggestion
(2022, 1, 0): various constants moved to enums, notably a controlTypes refactor.
```